src/services/weatherService.ts

```diff
@@ -2,7 +2,7 @@ import axios from 'axios';
 import { ReactNode } from 'react';
 
 const API_KEY = '2fd5ccd1fab6dcc6ec685dcec78345f7';
-const BASE_URL = 'https://api.openweathermap.org/data/3.0';
+const BASE_URL = 'https://api.openweathermap.org/data/2.5';
```

Comment:
this should go to .env file, if you don't have create it and  add this

https://create-react-app.dev/docs/adding-custom-environment-variables/